examples/flutter_gallery/lib/gallery/demo.dart

```diff
@@ -93,29 +97,28 @@ class TabbedComponentDemoScaffold extends StatelessWidget {
       child: Scaffold(
         appBar: AppBar(
           title: Text(title),
-          actions: (actions ?? <Widget>[])..addAll(
-            <Widget>[
-              Builder(
-                builder: (BuildContext context) {
-                  return IconButton(
-                    icon: const Icon(Icons.library_books, semanticLabel: 'Show documentation'),
-                    onPressed: () => _showApiDocumentation(context),
-                  );
-                },
-              ),
-              Builder(
-                builder: (BuildContext context) {
-                  return IconButton(
-                    icon: const Icon(Icons.code),
-                    tooltip: 'Show example code',
-                    onPressed: () => _showExampleCode(context),
-                  );
-                },
-              ),
-            ],
-          ),
+          actions: (actions ?? <Widget>[])..add(
```

Comment:
If we can use them, this would be a great place for the list conditionals.

something like

```dart
actions: <Widget>[
  ...actions
  Builder(/* params */), 
  if (showExampleCodeAction) 
    Builder(/* params */), 
]
```

Reply:
I tried to use that here but the version the gallery is on does not support them yet

Reply:
Ah, shame, thanks for trying it.